open-craft/opencraft
# Deploy new console

This PR merges the features implemented in multiple tasks related to the new frontend back into master.

**Setup instructions:**
1. Create a trial instance before deploying branch.
2. Wait for it to provision.
3. Deploy this branch, install requirements and apply migrations (take note of the migrations to revert later).

**Testing instructions:**
1. Check that the old trial instance provisions correctly from the new console. (but theming customizations aren't shown).
2. Create a new trial instance.
3. Check if it provisions correctly.
4. Apply customizations and deploy.
5. Wait for deployment and check if customizations were properly applied.
6. Take note of any bugs and add them to the epic.

**Reviewer:**

**Notes:**
UI bugs are not blockers for merging this. The goal here is to check if previous deployments are affected by this deployment + migrations.
